packages/firestore/test/unit/model/field_value.test.ts

```diff
@@ -485,4 +486,57 @@ describe('FieldValue', () => {
       }
     );
   });
+
+  it('estimates size correctly for fixed sized values', () => {
```

Comment:
These tests are pretty clever in that they don't encode the actual sizes, but they don't directly verify the implementation. It would be more straightforward if the tests just listed out:

```ts
function sizeof(value) {
  return wrap(value).approximateByteSize();
}

it('estimates size correctly for fixed sized values', () => {
  expect(sizeof(null)).to.be(4);
  expect(sizeof('')).to.be(0);
  // etc ...

});